packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3616,17 +3616,15 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       _bringIntoViewBySelectionState(oldTextSelection, value.selection, cause);
     }
     final String currentText = _value.text;
-    if (oldValue.text != currentText) {
-      try {
-        widget.onChanged?.call(currentText);
-      } catch (exception, stack) {
-        FlutterError.reportError(FlutterErrorDetails(
-          exception: exception,
-          stack: stack,
-          library: 'widgets',
-          context: ErrorDescription('while calling onChanged'),
-        ));
-      }
+    try {
```

Comment:
Should this check if `.text` OR `.composing` are different and only call onChanged then?

Reply:
Sorry, I thought the above code was judged, I have adjusted the code now.